src/material/tabs/_tabs-common.scss

```diff
@@ -343,6 +343,14 @@ $mat-tab-animation-duration: 500ms !default;
   flex-grow: 1;
   overflow: hidden;
   z-index: 1;
+  border-bottom-style: solid;
+  @include token-utils.use-tokens(
+      tokens-mat-tab-header.$prefix,
```

Comment:
nit: seems like a bit too much indentation here.

Reply:
Fixed - thanks. I'd expect formatters to catch this